events/models.py

```diff
@@ -1,13 +0,0 @@
-from django.db import models
-
-from locations.models import Location
-
-
-# Create your models here.
-class Event(models.Model):
```

Comment:
need to add class Meta
add relevant helping + error messages
add __str__ function for the admin